pkg/controller/cyclenoderequest/transitioner/transitions.go

```diff
@@ -557,3 +578,34 @@ func (t *CycleNodeRequestTransitioner) transitionSuccessful() (reconcile.Result,
 	t.rm.Logger.Info("Requeuing CycleNodeRequest for deleting later")
 	return reconcile.Result{Requeue: true, RequeueAfter: t.options.DeleteCNRRequeue}, nil
 }
+
+// transitionSkipped handles skipped CycleNodeRequests
+func (t *CycleNodeRequestTransitioner) transitionSkipped() (reconcile.Result, error) {
```

Comment:
there seems to be no difference between `transitionSkipped` and `transitionSuccessful`. Any reason not to use  `transitionSuccessful` directly ?